gr-fec/python/fec/bindings/python_bindings.cc

```diff
@@ -104,16 +106,18 @@ PYBIND11_MODULE(fec_python, m)
     bind_dummy_decoder(m);
     bind_dummy_encoder(m);
     bind_encode_ccsds_27_bb(m);
-    bind_fec_mtrx(m);
     // bind_gf2mat(m);
     // bind_gf2vec(m);
+#ifdef ENABLE_GSL
```

Comment:
```suggestion
#ifdef HAVE_GSL
```

Reply:
Thanks for your comments. I apply your proposals